src/appsco-access-report-page.html

```diff
@@ -0,0 +1,258 @@
+                margin-top: 10px;
+                overflow: auto;
+            }
+            :host .info-content > * {
+                height: 78px;
+            }
+            :host .access-report-items {
+                @apply(--layout-flex-none);
+            }
+            :host div[content] > * {
+                padding: 0 10px 10px 10px;
+            }
+            .access-report-container {
+                margin-top: 10px;
+            }
+            appsco-company-access-report-image {
+                --access-report-image-preview: {
+                    width: 32px;
+                    height: 32px;
+                    margin-right: 5px;
+                };
+
+                --access-report-image-background-color: var(--body-background-color-darker);
+            }
+            :host div[info] .info-header {
+                border-bottom: 1px solid var(--divider-color);
+                height: 50px;
+            }
+            :host .info-header .access-report-name {
+                @apply(--paper-font-subhead);
+                font-size: 18px;
+            }
+
+            :host div[info] .info-actions {
+                padding: 0;
+                position: absolute;
+                bottom: 0;
+                left: 0;
+                right: 0;
+            }
+            :host .edit-button {
+                @apply(--primary-button);
+                border-radius: 0;
+            }
+            :host .flex-horizontal {
+                @apply(--layout-horizontal);
+                @apply(--layout-center);
+            }
+            :host .flex-vertical {
+                @apply(--layout-vertical);
+            }
+            :host .flex {
+                @apply(--layout-flex);
+            }
+            :host([tablet-screen]) {
+                --info-width: 250px;
+            }
+            :host([mobile-screen]) {
+                --info-width: 100%;
+            }
+            :host([screen992]) {
+                --account-basic-info: {
+                    width: 140px;
+                };
+                --account-basic-info-values: {
+                    width: 140px;
+                };
+
+                --account-auth-type-info: {
+                    width: 140px;
+                };
+                --account-auth-type-info-values: {
+                    width: 140px;
+                };
+            }
+        </style>
+
+        <iron-media-query query="(max-width: 992px)" query-matches="{{ screen992 }}"></iron-media-query>
+        <iron-media-query query="(max-width: 800px)" query-matches="{{ tabletScreen }}"></iron-media-query>
+        <iron-media-query query="(max-width: 600px)" query-matches="{{ mobileScreen }}"></iron-media-query>
+
+        <appsco-content id="appscoContent">
+
+            <div content>
+                <div class="access-report-container">
+
+                </div>
+            </div>
+
+            <div class="flex-vertical" info>
```

Comment:
There shouldn't be Info section but Resource section (left section of the page). Please check screenshot on the task.